updates/stable.json

```diff
@@ -1,24 +1,24 @@
 {
   "stream": "stable",
   "metadata": {
-    "last-modified": "2020-01-14T19:32:23Z"
+    "last-modified": "2020-01-31T19:31:40Z"
   },
   "releases": [
     {
-      "version": "31.20200108.3.0",
+      "version": "31.20200113.3.1",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "31.20200113.3.1",
+      "version": "31.20200118.3.0",
       "metadata": {
         "rollout": {
-          "start_epoch": 1579032000,
+          "start_epoch": 1580738400,
           "start_percentage": 0.0,
-          "duration_minutes": 1440
+          "duration_minutes": 4320
```

Comment:
this is worth highlighting: 3 day rollout window

Reply:
since this is our first stable release upgrade, i like to be a little more conservative